no member named 'GetCocoaServer'

Hi;
When I compile openCV 3 (HEAD) version, it has been broken compiling.
I can use PCL and VTK normaly.
How can I solve this problem.

```
/tmp/opencv320160323-37738-fjug24/modules/viz/src/vtk/vtkCocoaInteractorFix.mm:284:81: error: no member named 'GetCocoaServer' in 'cv::viz::vtkCocoaRenderWindowInteractorFix'
        vtkCocoaServerFix *server = reinterpret_cast<vtkCocoaServerFix*> (this->GetCocoaServer ());
                                                                          ~~~~  ^
/tmp/opencv320160323-37738-fjug24/modules/viz/src/vtk/vtkCocoaInteractorFix.mm:285:20: error: no member named 'GetCocoaServer' in 'cv::viz::vtkCocoaRenderWindowInteractorFix'
        if (!this->GetCocoaServer ())
             ~~~~  ^
/tmp/opencv320160323-37738-fjug24/modules/viz/src/vtk/vtkCocoaInteractorFix.mm:288:19: error: no member named 'SetCocoaServer' in 'cv::viz::vtkCocoaRenderWindowInteractorFix'
            this->SetCocoaServer (reinterpret_cast<void*> (server));
            ~~~~  ^
/tmp/opencv320160323-37738-fjug24/modules/viz/src/vtk/vtkCocoaInteractorFix.mm:300:81: error: no member named 'GetCocoaServer' in 'cv::viz::vtkCocoaRenderWindowInteractorFix'
        vtkCocoaServerFix *server = reinterpret_cast<vtkCocoaServerFix*> (this->GetCocoaServer ());
                                                                          ~~~~  ^
4 errors generated.
make[2]: *** [modules/viz/CMakeFiles/opencv_viz.dir/src/vtk/vtkCocoaInteractorFix.mm.o] Error 1
make[1]: *** [modules/viz/CMakeFiles/opencv_viz.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/modules/viz/src/vtk/vtkCocoaInteractorFix.mm
+++ b/modules/viz/src/vtk/vtkCocoaInteractorFix.mm
@@ -160,7 +160,7 @@
 
 //----------------------------------------------------------------------------
 
-#if VTK_MAJOR_VERSION >= 6 && VTK_MINOR_VERSION  >=2
+#if VTK_MAJOR_VERSION >= 6 && VTK_MINOR_VERSION >=2 || VTK_MAJOR_VERSION >=7
 
 namespace cv { namespace viz
     {

